reposilite-plugins/webhook-plugin/src/main/kotlin/com/reposilite/plugin/webhook/api/DiscordWebhook.kt

```diff
@@ -0,0 +1,70 @@
+package com.reposilite.plugin.webhook.api
+
+// Core
+import club.minnced.discord.webhook.WebhookClientBuilder;
+// Embed
+import club.minnced.discord.webhook.receive.ReadonlyMessage;
+import club.minnced.discord.webhook.send.WebhookEmbedBuilder
+// Message
+import club.minnced.discord.webhook.send.WebhookMessageBuilder;
+import com.reposilite.maven.Repository
+import com.reposilite.storage.api.Location
+// Config
+
+
+
+fun sendDiscordEmbed(by: String, gav: Location, repo: Repository) {
```

Comment:
I think that Discord Webhook should implement some kind of an abstract interface/class offered by webhook domain API. It's related to the discussion on discord and the strategy pattern (so pretty much about providing an abstract layer that allows to implement multiple integrations using the same API)